README.md

```diff
@@ -1,169 +1,30 @@
-# XMTP-JS remote content types
+# XMTP JS Content Types
```

Comment:
Thank you for the clarification in this heading, Ry - removing "remote" makes sense as this repo will grow to hold more than the remote content type. So good.

```suggestion
# XMTP JS content types
```